.tekton/pull-request.yaml

```diff
@@ -52,7 +52,7 @@ spec:
             - name: run-tests
               image: registry.access.redhat.com/ubi8/ubi:latest
               workingDir: $(workspaces.workspace.path)
-              script: yum install -y make && make prepare
+              script: yum install -y make go && make prepare
```

Comment:
do we need a specific go version?

Reply:
I removed this installation. Not needed anymore. I'm not calling the dashboard linter from the `prepare_tests`